include/vapor/FlowParams.h

```diff
@@ -171,35 +246,101 @@ class PARAMS_API FlowParams : public RenderParams {
     //! The rendering type that represents the flow paths.
     //! See RenderType enum class.
     static const std::string RenderTypeTag;
+
     static const std::string RenderRadiusBaseTag;
-    //! Scales the radius of the flow tube rendering
+
+    //! Scales the radius of the flow tube rendering.
+    //! Applies data of type: double.
+    //! Typical values: 0.1 to 5.0.
+    //! Valid values: DBL_MIN to DBL_MAX.
     static const std::string RenderRadiusScalarTag;
-    //! Toggles between rendering 2d glyphs and 3d geometry of the render type
+
+    //! Toggles between rendering 2d glyphs and 3d geometry of the render type.
+    //! Applies data of type: bool.
+    //! Valid values: 0 = off, 1 = on.
     static const std::string RenderGeom3DTag;
+
     static const std::string RenderLightAtCameraTag;
-    //! Draws the direction of the flow stream
+
+    //! Draws the direction of the flow stream.
+    //! Applies data of type: bool.
+    //! Valid values: 0 = off, 1 = on.
     static const std::string RenderShowStreamDirTag;
     
-    //! When rendering samples, determines whether samples are rendered as circles or arrows
+    //! When rendering samples, determines whether samples are rendered as circles or arrows.
+    //! Applies data of type: long.
+    //! Valid values: 0 = FloatParams::GlyphTypeSphere, 1 = FloatParams::GlyphTypeArrow.
     static const std::string RenderGlyphTypeTag;
-    //! When rendering samples, draw every N samples
+
+    //! When rendering samples, draw every N samples.
+    //! Applies data of type: int.
+    //! Typical values: 1 to 20.
+    //! Valid values: INT_MIN to INT_MAX.
     static const std::string RenderGlyphStrideTag;
-    //! When rendering samples, only draw the leading sample in a path
+
+    //! When rendering samples, only draw the leading sample in a path.
+    //! Applies data of type: bool.
+    //! Valid values: 0 = off, 1 = on.
```

Comment:
Any reason why you've started using a different convention for specifying "Valid values" then earlier in this file? See, for example, GetSeedGenMode

Reply:
I felt that I should add more detail to the tags because they're just strings, and the reader can't infer what data type they correspond to.  Would you like me to add the "Valid values" and "Typical values" information to the setters and getters as well?

Reply:
OK. Would like to see how the documentation appears in the two cases: functions for which there is a C++ setter/getter, and functions for which there are not C++ setter/getters.